cluster/gce/util.sh

```diff
@@ -803,7 +806,8 @@ function construct-linux-kubelet-flags {
     flags+=" --non-masquerade-cidr=${NON_MASQUERADE_CIDR}"
   fi
   flags+=" --volume-plugin-dir=${VOLUME_PLUGIN_DIR}"
-  local node_labels="$(build-linux-node-labels ${node_type})"
+  local node_labels
+  node_labels=$(build-linux-node-labels "${node_type}")
```

Comment:
same here, I'm not convinced this is the same as before?
```suggestion
  node_labels="$(build-linux-node-labels ${node_type})"
```

Reply:
Quotes are superfluous on assignments.

```
echo 'foo bar' > foobar.txt
a=$(cat foobar.txt)
b="$(cat foobar.txt)"
```

will result in `a` and `b` having the same content, `foo bar`.

On the other hand, there are indeed added quotes around `${node_type}` to silence shellcheck complaining about the not being quoted. In cases where `node_type` would contain a whitespace, this would bring a different behavior, but there is no such cases, is there? (One option is of course a `#shellcheck disable` statement and a comment explaining the disable.)

Reply:
Re-added quotes according to previous discussion (in other PR).